applications/example/schemas/version.js

```diff
@@ -0,0 +1,3 @@
+{
+  version: 1,
+}
```

Comment:
It's ok for now but we need schema name, author, license, checksum in `version.js` and I'd rename it to `.schema` or something like `schema.pkg` @nechaido

Reply:
Perhaps ms.js?